modules/virt-creating-linux-bridge-nncp.adoc

```diff
@@ -3,15 +3,14 @@
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
 
 :_content-type: PROCEDURE
-[id="virt-configuring-linux-bridge-nncp_{context}"]
+[id="virt-creating-linux-bridge-nncp_{context}"]
 = Creating a Linux bridge using a node network configuration policy
 
-As a network administrator, you can create a Linux bridge interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+Use a `NodeNetworkConfigurationPolicy` manifest YAML file to create the Linux bridge.
```

Comment:
It's often a good idea to put the focus of the task first: 

You create a Linux bridge by applying a NodeNetworkConfigurationPolicy manifest to the cluster.

Reply:
Done.